src/pages/HomePage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import AddNewBook from '../components/Books/AddNewBook';
+
+import BookList from '../components/Books/BookList';
+import Card from '../components/UI/Card';
+
+const DUMMY_BOOKS = [
+  {
+    id: 1,
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    category: 'Action',
+  },
+  {
+    id: 1,
+    title: 'Dune',
+    author: 'Frank Herbert',
+    category: 'Science Fiction',
+  },
+  {
+    id: 1,
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+    category: 'Economy',
```

Comment:
- [ ] [OPTIONAL] finally the last one says we have the same key for different elements which should be unique :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213399356-d3135518-3212-4072-8700-1d5b71847248.png)

we can easily fix this by changing the key property to be unique for each book object :+1: